src/Core/AspectKernel.php

```diff
@@ -127,6 +144,16 @@ public function getContainer()
     }
 
     /**
+     * Returns an instance of cache path resolver
+     *
+     * @return null|CachePathResolver
+     */
+    public function getCachePathResolver ()
```

Comment:
This method violates SRP of `AspectKernel`. Should be removed. If some code wants this instance, please inject it directly via Dependency Injection principle in the container.